### ISSUE TITLE
Smart Profiler: Settings for Open Archive

### ISSUE THREAD

**Issue by cstiens** [guardianproject/viento#1]
![00 settings](https://user-images.githubusercontent.com/1434597/33976462-a5cad1b2-e05a-11e7-8902-7321682417a1.png)

![openarchive](https://user-images.githubusercontent.com/1434597/33976464-a7f9bf20-e05a-11e7-943c-e43d9ac41d0f.png)

**Comment by n8fr8** [comment]
For the first screen, shouldn't the different modes be selectable via a slider or radio buttons? How is that you can enable multiple at the same time?

Otherwise, thanks for getting these up!

**Comment by cstiens** [comment]
Yeah, you would only be able to select one mode at a time. So, if you enabled a different mode, the one that's currently on would turn off. Technically speak, radio buttons would be the most accurate. But, I like the experience of having toggles here, and I think we can get away with it. :)

**Comment by tiffrobo** [comment]
@n8fr8, could you help answer these questions we have about the current or intended workflow of OpenArchive as well as media compression?

1. When you upload to OA is it a duplicate file,  or mirroring what’s on your phone?
2. Currently is the file 'removed’ from your phone gallery /device? Or does the original stay in the phone gallery? Or both places? 
3. Compression:
    1. Where does compression happen? Is it on the server side or client side?
    2. What needs to be considered? (Bit rate, resolution, etc.)
    3. Do we (OA) want to set the parameters or allow the user to do it?
        1. GroupMe-limits video to 30sec
        2. WhatsApp-compresses files over 16MB and/or 640X360px
4. Storing only some in local storage, syncing the rest?

**Comment by eighthave** [comment]
This seems like a scary amount of preferences.  I guess that's OK in an "advanced" screen, as long as its not prominent.
